plugin/yt_dlp_plugins/extractor/getpot_bgutil_script.py

```diff
@@ -86,13 +113,12 @@ def _get_pot(self, client: str, ydl: YoutubeDL, visitor_data=None, data_sync_id=
             except json.JSONDecodeError as e:
                 raise RequestError(
                     f'Error parsing JSON response from _get_pot_via_script (caused by {e!r})') from e
-            else:
-                self._logger.debug(
-                    f'_get_pot_via_script response = {script_data_resp}')
-            if potoken := script_data_resp.get('poToken'):
-                return potoken
-            else:
+            if 'poToken' not in script_data_resp:
                 raise RequestError('The script did not respond with a po_token')
+            else:
```

Comment:
We can remove this `else` since the previous condition raises an exception early